Sample Applications/EditingExaminerDemo/MainWindow.xaml

```diff
@@ -202,7 +209,7 @@ Selection = RichTextBox.Selection;
                 </TabItem>
             </TabControl>
             <StackPanel Name="Panel3">
-                <Label Name="Label1" BorderThickness="1,0,1,0" BorderBrush="Gray" HorizontalContentAlignment="Center"
+                <Label Name="Label1" BorderThickness="1,0,1,0" Style="{StaticResource labelStyle1}" HorizontalContentAlignment="Center"
```

Comment:
Similar to the above comments by @arpitmathur , these should be more descriptive.